join_github_app/templates/pages/digital-justice-user.html

```diff
@@ -0,0 +1,48 @@
+{% extends "components/base.html" %}
+
+{% block pageTitle %}
+Using SSO for GitHub Access
+{% endblock %}
+
+{% block beforeContent %}
+{{ govukBackLink ({
+'text': 'Back',
+'href': '/select-organisations'
+}) }}
+{% endblock %}
+
+{% block content %}
+<h1 class="govuk-heading-l">Using Single Sign-On Access</h1>
+
+<p class="govuk-body">
+  Your email address <strong>{{email}}</strong> is registered for SSO access to the GitHub organisation. Single Sign-On
+  (SSO) is a convenient
+  method allowing you to access multiple services with one set of login credentials.
+</p>
+<p class="govuk-body">
+  By clicking the button below for the chosen organisation, you'll be directed to a secure login
+  page.
+</p>
+
+<p class="govuk-body">
+  Enter your @digital.justice.gov.uk credentials there, and you'll be automatically joined to the selected GitHub
```

Comment:
```suggestion
  Enter your @digital.justice.gov.uk credentials there to automatically join the selected GitHub
```
I think passive voice is also best avoided (aim for active, present)